packages/viewer/src/properties.ts

```diff
@@ -71,31 +81,34 @@ export class Properties {
       const urlParams = new URLSearchParams(params);
 
       if (urlParams.get('dpi') !== null && urlParams.get('dpi') !== undefined) {
-        instance.config.dpi = +urlParams.get('dpi');
+        Properties.instance.config.dpi = +urlParams.get('dpi');
```

Comment:
¿Por qué en el caso de la propiedad `zoom` y `dpi` se suman los valores a lo que ya hay en lugar de simplemente machacarlo?

Reply:
It is to convert the value to integer. We should change it to something clearer.